tcl.yaml

```diff
@@ -1,10 +1,13 @@
 package:
   name: tcl
   version: 9.0.1
-  epoch: 0
+  epoch: 1
   description: The Tcl scripting language
   copyright:
     - license: TCL
+  dependencies:
+    runtime:
+      - sqlite-libs
```

Comment:
Unrelated?

If you found this as a result of ldd-check, then you shouldn't need to add it (because melange sca would find it, right?)

Reply:
This was definitely missing.